server/config/requestHandler.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
   postSignin: function(req, res) {
     // check user exist in the database
     console.log('postSignin fired!', req.body);
-    var username = req.body.username.toLowerCase();
+    var username = req.body.username;
```

Comment:
@rahimftd We initially lowercased everything because we noticed our text inputs were autocapitalizing. 
Removed lowercase because we weren't lowercasing when we actually parsed usernames from the incoming form and I just disabled auto caps (see commits below) on all text inputs.

We can add back toLowerCase if we think it'll make for a better user experience.